scripts/Interface/ui/drill/model/DrillParameter.py

```diff
@@ -0,0 +1,222 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+
+import numpy
+
+from qtpy.QtCore import QObject, Signal
+
+from mantid.kernel import *
```

Comment:
Just an early comment, could you import only what you need? * imports are tolerated only for `simpleapi`, as it contains only functions.

Reply:
Done